web/styles.d.ts

```diff
@@ -2,3 +2,5 @@ declare module '*.css' {
   const content: any;
   export = content;
 }
+
+declare module '*.svg';
```

Comment:
Potentially add other asset types here in the future, `.jpg` `.png` for example.